docs/coding-standards/js.md

```diff
@@ -1,61 +1,164 @@
-# JavaScript
+# JavaScript style guide
+## Files
+JavaScript files have the same name as the component's folder name. Test files have a `.test` suffix placed before the file extension.
 
-# Linting
+```
+checkboxes
+├── checkboxes.js
+└── checkboxes.test.js
+```
+## Skeleton
 
-GOV.UK Frontend uses [standardjs](http://standardjs.com/), an opinionated JavaScript linter. All JavaScript files follow its conventions, and it runs on CI to ensure that new pull requests are in line with them.
+```js
+import { nodeListForEach } from '../globals/common'
 
-## Running standard manually
+function Checkboxes ($module) {
+  ...
```

Comment:
do we want to use `...` when [dots have meaning in javascript](https://dmitripavlutin.com/how-three-dots-changed-javascript/)? Maybe just something like `// code here`. Or I could be over-thinking it

Reply:
for me it felt easier to follow/less disruptive when reading, but makes sense. I will update with comments.

Reply:
@joelanman: would `// ...` be clear enough?